src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2310,10 +2314,11 @@ private static StringBuilder BuildCommandLine(string executableFileName, string
                 builder.Append('"');
             }
 
-            if (!string.IsNullOrEmpty(arguments))
+            foreach (var arg in arguments)
```

Comment:
Please add a comment here why escaping is needed here (i.e., this is Windows only, going through Win32 API...).